src/Tag.php

```diff
@@ -130,9 +130,37 @@ private function getTagInstance($tag)
 	public static function getTagClassName($tag)
```

Comment:
Is there a need for the function to be static? And public?

Reply:
Used in Columns.php

Reply:
Is than protected still useful?

Reply:
No, not really, when paradigm shifted from inheritance to composition. Can be handy in some cases, but common best practice is to avoid it. (If I understand your somewhat off-topic question correctly).

Reply:
sorry, for these not-expert (but still-interested) questions/remarks... mainly for my understanding about your design decisions...
I mean, if you like to limit the scope of the getTagClassName function, you could use it as protected static function. Because it is only used in Tag and inherited tags. 
A function that is protected can be only used here, so if you like to shift to another approach, you have only these classes that could actually use it. Therefore you know that you will break nothing by removing it at that time.